### DISCUSSION TITLE
BUG! cannot update draft

### DISCUSSION THREAD

**Discussion by blueMountain007** [payloadcms/payload#1251] [Q&A]
### After the update operation, the document changed from draft state to published, which is not what I expected, I just want to update the fields

[draft doc](https://payloadcms.com/docs/versions/drafts#database-changes)

```
await payload.update({
  draft: true,
  collection: 'articles',
  id: xxx,
  data: {
    migratedFromContentful: false
  },
});
```

<img width="1026" alt="image" src="https://user-images.githubusercontent.com/16400882/195586694-a59acc6f-9bb8-48bf-a75c-4b42f8c962b8.png">

**Comment by jacobsfletch** [comment]
Hey @zhounewz is this still an issue for you? Your request looks right to me, I see that you're setting `draft: true` just as the docs indicate. What version of Payload are you on? Try bumping to latest and let me know if this persists.
